docker/php/Dockerfile

```diff
@@ -41,4 +43,27 @@ RUN mv drush /usr/local/bin
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
 RUN composer --version
 
+# Install and setup XDebug.
```

Comment:
Can you do something like this: https://github.com/expresso-php/expresso-php/pull/67/files#diff-4e5e90c6228fd48698d074241c2ba760R28

Reply:
XDEBUG
Uncomment this section to enable XDEBUG.
....
END XDEBUG